core/src/main/resources/error/error-classes.json

```diff
@@ -1209,6 +1226,23 @@
     ],
     "sqlState" : "22032"
   },
+  "INVALID_LAMBDA_FUNCTION_CALL" : {
+    "message" : [
+      "Invalid lambda function call."
+    ],
+    "subClass" : {
+      "ARGUMENTS_NUMBER_MISMATCH" : {
+        "message" : [
+          "The number of lambda function arguments '<namesSize>' does not match the number of arguments expected by the higher order function '<argInfoSize>'."
+        ]
+      },
+      "ARGUMENT_NAMES_EXISTS_SAME_SEMANTIC" : {
+        "message" : [
+          "Lambda function arguments should not have names that are semantically the same."
```

Comment:
Could you make the error message clear and more precise. From the implementation:
```
    if (!conf.caseSensitiveAnalysis) {
      // scalastyle:off caselocale
      s: String => s.toLowerCase
      // scalastyle:on caselocale
    } else {
      s: String => s
    }
``` 

Like:
```
A lambda function has duplicate arguments: <dupArgs>. Consider to rename the arguments or to set the SQL config <caseSensitiveConfig> to \"false\".
```

Reply:
I don't get your point yet.
I guess you mean is `to set the SQL config <caseSensitiveConfig> to \"true\"`.